app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+
+  def create
+
+    if user = User.from_omniauth(request.env["omniauth.auth"])
```

Comment:
what's the sad path here. what happens if you don't get the user from github? what should a user see?